src/providers/realtime-provider.js

```diff
@@ -248,6 +272,15 @@ export default class RealtimeProvider {
                 } else if (this.isEventMessage(message)) {
                     const datum = eventToTelemetryDatum(message.data);
                     subscriptionDetails.callback(datum);
+                } else if (this.isMdbChangesMessage(message)) {
+                    let alarmRange = message.data.parameterOverride.defaultAlarm?.staticAlarmRange;
+                    let datum;
+                    if (alarmRange) {
+                        datum = getLimitFromAlarmRange(alarmRange)
```

Comment:
You may be able to get rid of the if/else if you assign it above like: `let alarmRange = message.data.parameterOverride.defaultAlarm?.staticAlarmRange ?? [];`

Reply:
Excellent idea.